### DISCUSSION TITLE
Error while adding route path to a new page

### DISCUSSION THREAD

**Discussion by Not-Sarthak** [refinedev/refine#4828] [Q&A]
I was exploring refine dev via the tutorial made by JSMastery but unable to understand where can I add a Route path to a new page. Wherever I try to add a Path via <Route> tag as in simple ReactJS it does not set the path. I did try using the <Link> Tag to but yet unable to set path for a new page. I tried reading the documentation but couldn't find a solution as the Routes are already set by JSMastery which I found out a bit tough to understand.

**Comment by Not-Sarthak** [comment]
Ok so the path is set but the page contains the following text "404 Not Found" and anything I add in the component is added above this text. How can I remove it? I am a beginner at this and it would be helpful if someone could help me out with this

**Comment by BatuhanW** [comment]
Hello @Not-Sarthak please read our documentation about router and double check your code. https://refine.dev/docs/packages/documentation/routers/react-router-v6/

**Reply by Not-Sarthak** [reply]
Hey, I did read the documentation but did not find any specific solution related to my issue.
